website/models.py

```diff
@@ -7,4 +7,24 @@ class User(models.Model):
     last_name = models.CharField(max_length=64)
     email_address = models.CharField(max_length=254)
     password = models.CharField(max_length=64)
-    salt = models.CharField(max_length=64)
\ No newline at end of file
+    salt = models.CharField(max_length=64)
+
+
+class Notification(models.Model):
+    id = models.CharField(max_length=64)
+    name = models.CharField(max_length=64)
+    notification_group = models.CharField(max_length=64)
```

Comment:
It's actually both, with the way that Django works.

When you create the models, Django creates database migrations which create
the tables based on the models you have created. You can check them out in
the migrations folder.

Actually, every time you add a model or modify one you will need to run

manage.py makemigrations (This creates a new migration with the new
models/changes)
manage.py migrate (This applies the migration and creates the appropriate
tables)

Does that make sense?

On Tue, Sep 29, 2015 at 7:43 AM Adam Thornton <notifications@github.com>
wrote:

> In website/models.py
> <https://github.com/justKnoWhere/knowhere/pull/3#discussion_r40665743>:
>
> > \ No newline at end of file
> > +    salt = models.CharField(max_length=64)
> > +
> > +
> > +class Notification(models.Model):
> > +    id = models.CharField(max_length=64)
> > +    name = models.CharField(max_length=64)
> > +    notification_group = models.CharField(max_length=64)
>
> Guess I'm confused. Are these models for the database or the UI? I was
> assuming the UI but it sounds like you are saying this is for the database.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/justKnoWhere/knowhere/pull/3/files#r40665743>.
>